core/templates/pages/exploration-player-page/layout-directives/audio-bar.directive.html

```diff
@@ -252,6 +252,11 @@
     top: 3.5em;
   }
 
+  @media(max-width: 768px) {
+    .headroom--unpinned.audio-header.audio-header-margin {
+      margin-top: 8px;
```

Comment:
Hm, were you able to figure out which PR caused this regression? I think this was a recent regression. We need to figure out what broke this in the first place to determine an optimal way to fix this.

Reply:
I have just added audio-player position while scrolling .It had absolute positioning before causing this bug.